gateway/tx.go

```diff
@@ -95,9 +94,10 @@ func postRequestGuardHandlerFn(rs *RestServer) http.HandlerFunc {
 			return
 		}
 
-		signedSimplexStateBytes := mainchain.Hex2Bytes(req.SignedSimplexStateBytes)
-		msg := subscribe.NewMsgRequestGuard(req.EthAddr, signedSimplexStateBytes, transactor.CliCtx.GetFromAddress())
-		writeGenerateStdTxResponse(w, transactor, msg)
+		// FIX
```

Comment:
could you please add a comment on fix what, same for others

Reply:
discuss offline, this will be fixed soon in next PR. Therefore, I will not add extra comments here